data/content_scripts/posting_process/iframe.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileOverview Script used to post the privly URL into the Host page target node.
```

Comment:
Hmm, You've got me thinking about this script. I don't think it's needed. I'll make the necessary changes.